src/lib.rs

```diff
@@ -405,7 +406,12 @@ pub fn workspaces(module: &mut Module<i64>) -> String {
 pub fn network(_data: &mut Module<()>) -> String {
     // Read the operstate file to see if the wireless is up
     let status_path = String::from(WL_PATH) + "operstate";
-    let status = read_to_string(status_path).expect("Failed to read wireless status");
+    let status = read_to_string(status_path);
+    if status.is_err() {
+        return paint("ERROR", ERROR_COLOR, "F");
```

Comment:
```suggestion
        return paint("ERROR: wireless status could not be read", ERROR_COLOR, "F");
```
Just having ERROR is not a very clear indication of what is wrong.